Spark/metrics.csv

```diff
@@ -0,0 +1,207 @@
+spark.stage.disk_bytes_spilled.count,rate,,byte,,The number of times the number of bytes in disk spilled in the application's stages was sampled,0,spark,stg dsk byt spld cnt
+spark.driver.rdd_blocks,rate,,block,,The number of RDD blocks in the driver,0,spark,dvr rdd blk
+spark.driver.memory_used,rate,,byte,,The amount of memory used in the driver,0,spark,dvr mem usd
+spark.driver.disk_used,rate,,byte,,The amount of disk used in the driver,0,spark,dvr dsk usd
+spark.driver.active_tasks,rate,,task,,The number of active tasks in the driver,0,spark,dvr act tsk
+spark.driver.failed_tasks,rate,,task,,The number of failed tasks in the driver,0,spark,dvr fld tsk
+spark.driver.completed_tasks,rate,,task,,The number of completed tasks in the driver,0,spark,dvr comp tsk
+spark.driver.total_tasks,rate,,task,,The number of total tasks in the driver,0,spark,dvr tot task
+spark.driver.total_duration,rate,,millisecond,,The driver's total duration,0,spark,dvr tot dur
+spark.driver.total_input_bytes,rate,,byte,,The number of input bytes in the driver,0,spark,dvr tot in byt
+spark.driver.total_shuffle_read,rate,,byte,,The total number of bytes read during a shuffle in the driver,0,spark,dvr tot shfl rd
+spark.driver.total_shuffle_write,rate,,byte,,The total number of bytes written durin a shuffle in the driver,0,spark,dvr tot shfl wrt
+spark.driver.max_memory,rate,,byte,,The maximum memory used in the driver,0,spark,dvr max mem
+spark.executor.count,rate,,,,The number of executors in the Spark application,0,spark,exe cnt
+spark.executor.rdd_blocks.max,gauge,,block,,The max number of RDD blocks in the application's executors,0,spark,exe rdd blk mx
+spark.executor.rdd_blocks.avg,gauge,,block,,The average number of RDD blocks in the application's executors,0,spark,exe rdd blk avg
+spark.executor.rdd_blocks.median,gauge,,block,,The median number of RDD blocks in the application's executors,0,spark,exe rdd blk med
+spark.executor.rdd_blocks.95percentile,gauge,,block,,The 95th percentile number of RDD blocks in the application's executors,0,spark,exe rdd blk 95
+spark.executor.rdd_blocks.count,rate,,block,,The number of times the number of RDD blocks in the application's executors was sampled,0,spark,exe rdd blk cnt
+spark.executor.memory_used.max,gauge,,byte,,The max amount of memory used in the application's executors,0,spark,exe mem usd mx
+spark.executor.memory_used.avg,gauge,,byte,,The average amount of memory used in the application's executors,0,spark,exe mem usd avg
+spark.executor.memory_used.median,gauge,,byte,,The median amount of memory used in the application's executors,0,spark,exe mem usd med
+spark.executor.memory_used.95percentile,gauge,,byte,,The 95th percentile amount of memory used in the application's executors,0,spark,exe mem usd 95
+spark.executor.memory_used.count,rate,,byte,,The number of times the amount of memory used in the application's executors was sampled,0,spark,exe mem usd cnt

```

Comment:
For `spark.executor.memory_used` metrics let's specify that this is the memory used for cached RDD blocks.